docs/addon-kubeflow.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: docs
+title: "Kubeflow addon"
+permalink: /docs/addon-kubeflow
+---
+
+<span class="p-label--new">New in 1.16.2 </span>
```

Comment:
Same here, we should not point the users to the 1.16 track, kubeflow will not be there. Lets do "New in 1.17, also available on the latest track".

Reply:
Ok